### DISCUSSION TITLE
Next Features

### DISCUSSION THREAD

**Discussion by cschaerfe** [openpharma/visR#198] [Ideas]
We started out implementing a few functions that are important in our daily work for visualising time to event analysis. However, this is not where the package should stop. Some ideas from discussions with colleagues and stakeholders for additional visualisations to be added are listed below. Please comment with other ideas or open an issue and start implementing if you wish to take one on.

**Cohort Selection and Baseline Characteristics** 
- [x] Table 1
- [x] cohort attrition flowchart 
- [ ] cohort attrition barplot
- [ ] tables of disease characteristics or prior therapies

**Change from Baseline**
- [ ] Scatterplots incl. diff options for overlapping data points like jitter
- [ ] Waterfall plots
- [ ] Cumulative distribution plots

**Survival/Time to Event Analysis**
- [x] Kaplan-Meier plots (different defaults for descriptive vs. comparative questions)
- [x] KM Summary tables
- [ ] Forest Plot (@bailliem is working on this) 

**Treatment/Testing Pattern Analysis**
- [ ] Frequency Table of selected drugs by line
- [ ] Lineplot of year of start of 1st line vs % of patients by drug
- [ ] Stacked boxplot of types of line of therapy by year of diagnosis
- [ ] Donut plot showing distribution of cohort by max LOT with bars specifying most frequent drugs in each line
- [ ] Swimmer Plot showing patterns of treatment (or other activity) by patient

**Treatment Sequences**
- [ ] Flowchart specifying patient flow incl. Death, lost to follow up, and progression to next line 
- [x] Sankey plot (also showing numbers in non-interactive mode)
- [ ] Sunburst plot showing progression of drugs in lines

[a comment by an automated account is not shown]

**Comment by cschaerfe** [comment]
Discussion still open

**Comment by SHAESEN2** [comment]
Can we add this to our wiki or a more central document?
